quadratic/quadratic.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main(){
+	var a, b, c float64
+	fmt.Println("The program finds solutions of quadratic equation.Enter the coefficients A, B and C: ")
+	fmt.Scan(&a,&b,&c)
```

Comment:
There is a bug in a program. Check the example `a, b, c = 4, -2, 1`.

Reply:
Good catch.

But there is still a bug. It should be `x1 := (-b - math.Sqrt(d)) / (2 * a)`. `(2 * a)` must be in parenthesis. Same for `x2`.